ConcernsCaseWork/Service.TRAMS/Decision/DecisionService.cs

```diff
@@ -30,6 +31,18 @@ public async Task<CreateDecisionResponseDto> PostDecision(CreateDecisionDto crea
 			_logger.LogInformation($"Decision created. caseUrn: {postResponse.ConcernsCaseUrn}, DecisionId:{postResponse.DecisionId}");
 			return postResponse;
 		}
+
+		public async Task<List<GetDecisionResponseDto>> GetDecisionsByCaseUrn(long urn)
+		{
+			_logger.LogInformation($"{nameof(DecisionService)}::{LoggingHelpers.EchoCallerName()}");

```

Comment:
There is another extension method on Logger which logs method has been entered. Just a comment, not blocking the PR for this